src/containters/Home/components/HeroView/HeroView.js

```diff
@@ -1,39 +1,59 @@
-import React from 'react'
-import './index.scss'
-import { IoIosArrowForward } from 'react-icons/io'
-
+import React from 'react';
+import './index.scss';
+import { IoIosArrowForward } from 'react-icons/io';
+import { useEffect, useState } from 'react';
+import { heroView } from '../../services/heroView';
+import { BrowserRouter as Router, Link } from 'react-router-dom';
 const HeroView = () => {
+    const [heroViewData, setHeroViewData] = useState({});
```

Comment:
- Please move this functionality of fetching the data and handling it in local state  to **the outer container component** which is the **Home component**.
as it is common and more logic to make data fetching  inside container components and pass the fetched data as **props** to the child functional component which is the **HeroView** component